### ISSUE TITLE
Shortcut thumbnail aspect ratio

### ISSUE THREAD

**Issue by CristobalManga** [hrydgard/ppsspp#19330]
### What should happen

First of all, I am very happy that they have implemented the automatic thumbnail in the shortcuts on Android, I thank them very much.

But I wonder if it is possible for the miniature to have the same aspect ratio as the original miniature in the game.

Personally I think it will be more visually aesthetic.

But I don't know what the developers think about this.

[
![2024_07_16_20_35_40](https://github.com/user-attachments/assets/d728914d-03e7-4488-9c75-70d82f76bb11)
![2024_07_16_20_34_55](https://github.com/user-attachments/assets/5463e3db-75c3-418f-a328-4559230abac8)
](url)

### Who would this benefit

I think it would look better in an app drawer

### Platform (if relevant)

Android

### Games this would be useful in

All

### Other emulators or software with a similar feature

Duckstation

### Checklist

- [X] Check the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by hrydgard** [comment]
Is it even possible to have non-square icons on Android? I guess we could shrink them to fit, but they'd be very small...

**Comment by CristobalManga** [comment]
Icons can have any Android shape. The size of the icon will be defined by the user's settings in an app drawer.

Certainly the icon will look smaller in its vertical state, but that is what is original.

Furthermore, the minority of people who will use this function have been seeing the same icon on ppsspp or psp for a long time and will easily know the game they are looking for.

Actually in the following screenshot, the cover in its original state is not so small.
![2024_07_17_11_03_17](https://github.com/user-attachments/assets/f4260f67-1e8b-4c64-9e1c-c0a4abe690ad)

**Comment by hrydgard** [comment]
Yeah, given the size of android screens these days, maybe that's acceptable.
